packages/web3-wrapper/src/web3_wrapper.ts

```diff
@@ -403,8 +421,18 @@ export class Web3Wrapper {
         }
         return receipt;
     }
+    /**
+     * Sets the current head of the local chain by block number. Note, this is a
+     * destructive action and may severely damage your chain. Use with extreme
+     * caution.
```

Comment:
Let's mention to JSON RPC action name: `debug_setHead`. Does Parity also implement this? If not, we might want to mention that it's Geth specific, maybe even throw if backing node is not Geth?

Reply:
It will throw the error like: 'debug_setHead is not supported'

Reply:
👍 I added some clarity to the comment.

We already will throw if the backing provider doesn't support the `debug_setHead` endpoint. I think this is sufficient. Furthermore, what we do now is a form of feature detection (does this provider support this feature?) as opposed to specifically detecting the provider type (is this provider Geth?). The former is generally preferable if possible because it allows for adapting to future changes. If Parity or another node adds support in the future, we would incorrectly be throwing in the latter case.

Reply:
Makes sense! Great!